src/pages/ResearcherProfile.js

```diff
@@ -622,7 +622,7 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
               div({ className: 'form-group' }, [
                 div({ className: 'col-xs-12', style: { 'marginTop': '20px' } }, [
                   label({ className: 'control-label rp-title-question default-color' }, [
-                    'Researcher Identification*',
+                    'Researcher Identification',
```

Comment:
I think we should revert this change as well. The explanation text does a good job of explaining what is required and what isn't.